Makefile

```diff
@@ -6,6 +6,8 @@ PKG:=github.com/aws/aws-cloud-map-mcs-controller-for-k8s/pkg/version
 IMG ?= controller:latest
 # Produce CRDs that work back to Kubernetes 1.11 (no version conversion)
 CRD_OPTIONS ?= "crd:trivialVersions=true,preserveUnknownFields=false"
+# AWS Region
+AWS_REGION ?= us-west-2
```

Comment:
Maybe we should use `us-east-1` as the default region to be aligned with AWS SDK.